docs/prerequisites.rst

```diff
@@ -81,6 +81,7 @@ With Ubuntu 16.04, create a systemd init script at
 
 and start it with: ::
 
+  sudo chmod 600 /var/lib/mongodb/
```

Comment:
Is there a rationale for removing all permissions from other users of the system? If not, I'd suggest just

    sudo chmod u+rw /var/lib/mongodb/

Also, does this need to be applied recursively? If so add `-R` to the command.